pinc/TallyBoard.inc

```diff
@@ -476,7 +524,7 @@ class TallyBoard
         $sql = sprintf(
             "
             SELECT MAX(timestamp) as max_timestamp
-            FROM past_tallies
+            FROM latest_snapshot_tallies
```

Comment:
If we have `tally_snapshot_times`, it would be simpler to get the time from there.

Reply:
I thought about that, but this is more accurate since, as you've pointed out, taking snapshots isn't instantaneous.

Reply:
The `timestamp` that goes into `past_tallies` or `latest_snapshot_tallies` is `$ascribe_time`, which is set in `crontab/take_tally_snapshots.php`, and is constant for a given day's snapshot job. The fact that snapshots aren't instantaneous doesn't affect it.

Side comment: I'm not seeing where a new timestamp is added to `tally_snapshot_times` (outside of the upgrade scripts).